baus/ual.py

```diff
@@ -956,6 +973,26 @@ def hlcm_owner_simulate_no_unplaced(households, residential_units,
                          "price_equilibration")
 
 
+@orca.step()
+def hlcm_owner_lowincome_simulate_no_unplaced(households, residential_units,
+                                              year, final_year,
+                                              aggregations, settings,
+                                              hlcm_owner_lowincome_config,
+                                              hlcm_owner_lowincome_no_unplaced_config):
+
+    # only run in the last year, but make sure to run before summaries
+#    if year != final_year:
+#        return
+
+    drop_tenure_predict_filters_from_yaml(
+        hlcm_owner_lowincome_config,
+        hlcm_owner_lowincome_no_unplaced_config)
+
+    return hlcm_simulate(households, residential_units, aggregations,
+                         settings, hlcm_owner_lowincome_no_unplaced_config,
+                         "price_equilibration")
+
```

Comment:
Seems like we also need to run `update_unit_ids()` here again? Also in the other "no_unplaced" function.

Reply:
The comment placement got a little confusing here- but see my comment directly above this re: the "no_unplaced" function.

Reply:
Sorry I didn't see that! Looks good.